test/e2e/support.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable ui-testing/no-disabled-tests */
```

Comment:
I don't see any disabled tests. Is this rule buggy?

Reply:
Removed it